server/shared/content-plugins/elementRegistry.js

```diff
@@ -60,7 +61,7 @@ class ElementsRegistry extends BaseRegistry {
 module.exports = new ElementsRegistry();
 
 function deprecateHandleStatics(element) {
-  const name = `tce-${element.type.toLowerCase()}`;
+  const name = `tce-${toCase.slug(element.type)}`;
```

Comment:
@underscope are we ok with this or should we use the same logic with question-specific type resolution as in `getComponentName` from `client/components/common/tce-core/utils`?